### ISSUE TITLE
Problem with installing the libraries.

### ISSUE THREAD

**Issue by Monses34** [bdureau/RocketMotorPIDGimbal#1]
Is there a way to load the whole project onto Arduino?

**Comment by bdureau** [comment]
Yes, you need first to install the stm32 arduino board so that you can compile it.
you might be able to use other Arduino board but you might have to modify the code.
So first let me know which board you are using.

**Comment by neilferreri** [comment]
@bdureau 
Do you have a link to the `BMP085_stm32.h` source?
I couldn't compile with the adafruit library either.
Thanks!

**Comment by bdureau** [comment]
Yes you can find it here

https://github.com/bdureau/AltimetersLibs

 �
